.github/actions/BuildCirct/action.yml

```diff
@@ -4,17 +4,18 @@ description: "Builds CIRCT, which is used for the HLS backend"
 runs:
   using: "composite"
   steps:
+    - uses: actions/checkout@v4
+
```

Comment:
Why is this all in a sudden needed here?

Reply:
This is a copy paste mistake from the example I used. I'll remove it.

Reply:
Removed